README.md

```diff
@@ -1,7 +1,12 @@
-This repo is a proxy to https://github.com/powershell/powershell for creating on-demand packages when releasing PowerShellCore.
+The repo contains the infrastructure to bulid PowerShell Core in Docker.
 
-The Jenkinsfile at the root of the repo provides directives to build and package on all supported platforms.
+The module dockerBasedBuild is the main infrastructure.  The rest of the files are legacy.
 
-The repo also has scripts to build on specific platforms.
+Note:  This is currently highly geared toward VSTS, but we are open to contribution which make it more pluggable.  The majority of the VSTS specific operations have already been seperated into a separate module.
+
+To use this module:
+
+1. Create a wrapper script in your repo.  See docs/examples/vstsBuild.ps1.
```

Comment:
Why do you put the dockerfiles and scripts used for release build in `docs/examples`? Are we not going to use them for our release build?

Reply:
These files should be created in the product repo.  In our case, the PowerShell repo,  that was PowerShell owns its build definition.